sdks/python/apache_beam/transforms/stats_test.py

```diff
@@ -89,6 +90,12 @@ def setUp(self):
           None,
           0.1,
           'assert:global_by_error_with_large_population'),
+      (
+          'numpy_input_data',
+          np.array(range(10)),
```

Comment:
Do multidimensional arrays also work?

Reply:
No. For numpy input I am assuming that for ApproxUniqueCombineFn.add_input(self, accumulator, element), element is a numpy scalar (not a numpy array). 

However, if the element is a multidimensional numpy array, then calling element.item(0) returns the value at the first index of the array as a python type.

Reply:
I can allow multidimensional numpy arrays as input if you want.

Reply:
I see. At this point I am curious about how users use  ApproximateUnique. 
The use-case you are addressing is when users pass a PCollection of elements, where each element is a single value stored in a numpy datatype. Since it's a single value,  we convert it to a scalar. Is that right?

I am wondering if there is also a use-case when users pass a PCollection of numpy arrays (perhaps erroneously). In which case the current combiner will pick the first element of the array, so approximation may not be very precise. I wonder if a more precise implementation makes sense or this use-case is not common.

Reply:
Ah sorry, just saw this. I didn't like the assumption I made about the type of elements in the PCollection either, and the use case you mention is definitely unsupported with this draft edit but would not alert the user if they made that mistake.